### PR TITLE
Fixes for projectInputs and accessDataType

### PR DESCRIPTION
**Pull request by CeresBarros**

### CODE REVIEW

**Comment by achubaty** [comment]
It looks like it's failing because you aren't importing `getValues`.

File: R/postProcess.R [hunk 1]
```diff
@@ -807,17 +807,19 @@ assessDataType <- function(ras) {
 #' @export
 #' @rdname assessDataType
 assessDataType.Raster <- function(ras) {
-  minVal <- ras@data@min              ## using ras@data@... is faster than using raster functions
+  ## using ras@data@... is faster, but won't work for @values in large rasters
+  rasVals <- getValues(ras)
```
Comment (achubaty) [inline R/postProcess.R]: you'll need to add the follwing at line 799:

```r
#' @importFrom raster getValues
```

Comment (achubaty) [reply R/postProcess.R]: and be sure to rebuild the documentation afterward!

File: R/postProcess.R [hunk 2]
```diff
@@ -794,9 +794,10 @@ writeOutputs.default <- function(x, filename2, ...) {
 #'
 #' @param ras  The RasterLayer or RasterStack for which data type will be assessed.
 #' @author Eliot McIntire
-#' @author CeresBbarros
+#' @author CeresBarros
```
Comment (achubaty) [inline R/postProcess.R]: space?

File: tests/testthat/test-prepInputs.R [hunk 3]
```diff
@@ -989,6 +989,7 @@ test_that("load rdata in prepInputs", {
 
 
 test_that("assessDataType doesn't work", {
+  require(raster)
```
Comment (achubaty) [inline tests/testthat/test-prepInputs.R]: use `library()` in tests, and detach the package on exit (see other tests).

Comment (eliotmcintire) [reply tests/testthat/test-prepInputs.R]: Ideally use:
```
      testInitOut <- testInit(libraries = "raster")
      on.exit({
        testOnExit(testInitOut)
      }, add = TRUE)
```
See many examples. This function (`testInit`) loads the named libraries but also turns off "ask" and "verbose" and has options for other things. With `testOnExit`, it cleans up everything that changed (unloads libraries, resets all changed options).

**Comment by CeresBarros** [comment]
@Alex, anything else that I need to do here?

**Comment by achubaty** [comment]
nope, Eliot already merged. thanks!

**Pull request merged**